packages/data/lib/repository/app_version_repository.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:domain/model/app_version_model.dart';
+import 'package:domain/repository/app_version_repository.dart';
+import 'package:package_info_plus/package_info_plus.dart';
+
+const _versionCollection = 'star_movie_version';
+const _minVersion = 'min';
+const _actualVersion = 'actual';
+
+class AppVersionRepositoryImpl implements AppVersionRepository {
+  final FirebaseFirestore firebaseFirestore;
+  final PackageInfo packageInfo;
```

Comment:
u can use packageInfo in usecase, no need to have it in usecase.  +make fields private plz

Reply:
I didn’t understand a little) use it in usecase, but isn’t it necessary to use it there? I can't get your point)

Reply:
updated, sorry